### PR TITLE
Add types for multiple vigour.io projects

### PR DESCRIPTION
**Pull request by ffflorian**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

Included projects:
* [is-number-like](https://github.com/vigour-io/is-number-like)
* [quick-hash](https://github.com/vigour-io/quick-hash)
* [turbostatus](https://github.com/vigour-io/turbostatus)
* [vigour-ua](https://github.com/vigour-io/ua)

### CODE REVIEW

File: types/is-number-like/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+// Project: https://github.com/vigour-io/is-number-like#readme
+// Definitions by: Florian Keller <https://github.com/ffflorian>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/**
+ * Checks whether provided parameter looks like a number
+ * @param val the value to check
+ * @returns `true` if `val` looks like a number, `false` otherwise
+ */
+declare function isNumberLike(val: any): val is number;
```
Comment (RyanCavanaugh) [inline types/is-number-like/index.d.ts]: This function returns `true` for many values that aren't numbers

**Review by RyanCavanaugh — CHANGES_REQUESTED** [review]

**Comment by ffflorian** [comment]
@RyanCavanaugh Do you know why the types for `vigour-ua` and `turbostatus` were not published?

**Comment by RyanCavanaugh** [comment]
@sandersn investigate?

**Comment by sandersn** [comment]
All the packages published just fine, but github gave some kind of error during the comment-posting process. The comment-posting code gives up after any error.

Right now the publisher doesn't retry comment posting because
1. it's complicated, involving multiple calls to github.
2. it's slow-ish, and slows down publication for everything to retry multiple times.

On the other hand, everybody's now used to this feature, and they regard it as a failure when the comment fails, even when npm publishing succeeds.

**Comment by sandersn** [comment]
Filed https://github.com/microsoft/types-publisher/issues/618 to track the comment failure.

**Pull request merged**